app/Http/Controllers/WebpayPlusMallDeferredController.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use Transbank\Webpay\Options;
+use Transbank\Webpay\WebpayPlus;
+
+class WebpayPlusMallDeferredController extends Controller
+{
+    public function __construct(){
+        if (app()->environment('production')) {
+            WebpayPlus::setCommerceCode(config('services.transbank.webpay_plus_mall_deferred_cc'));
+            WebpayPlus::setApiKey(config('services.transbank.webpay_plus_mall_deferred_api_key'));
+            WebpayPlus::setIntegrationType('LIVE');
+        } else {
+            WebpayPlus::setCommerceCode("597055555544");
+            WebpayPlus::setApiKey("579B532A7440BB0C9079DED94D31EA1615BACEB56610332264630D42D0A36B1C");
+            WebpayPlus::setIntegrationType('TEST');
```

Comment:
This is not needed. The entire else :)

Reply:
Are you sure about that ? SDK does not come preconfigured for MallDeferred